roles/rsyslog/tasks/main.yml

```diff
@@ -90,9 +92,28 @@
             state: absent
             path: "{{ __rsyslog_config_dir }}/*"
           when: __rsyslog_purge_original_conf | bool | d(false)
+
+        - block:
+            - name: Get backup files
+              shell: ls "{{ __rsyslog_backup_dir }}"
```

Comment:
use `command` instead of `shell`?

Reply:
If there are no backups, what happens?

Reply:
> use `command` instead of `shell`?

:p Thanks!

> If there are no backups, what happens?

The backup dir is guaranteed to exist.
https://github.com/linux-system-roles/logging/blob/master/roles/rsyslog/tasks/main.yml#L73-L76

And if no backup files exist, extracting from the backup file is skipped.
https://github.com/nhosoi/logging/blob/restore_confs/roles/rsyslog/tasks/main.yml#L110